.github/workflows/check_release_name_bump.yml

```diff
@@ -0,0 +1,41 @@
+name: Check release bump
+
+env:
+  TZ: Europe/London
```

Comment:
Do we need to set the timezone in the container?

Reply:
not needed, will remove